packages/ui/src/theme/tokens/colors.ts

```diff
@@ -97,6 +97,30 @@ export type WebColors = {
   [key: string]: WebColorTypes | Record<string, WebColorTypes>;
 };
 
+export type ReactNativeColors = {
+  red: Record<string, string>;
+  orange: Record<string, string>;
+  yellow: Record<string, string>;
+  green: Record<string, string>;
+  teal: Record<string, string>;
+  blue: Record<string, string>;
+  purple: Record<string, string>;
+  pink: Record<string, string>;
+  neutral: Record<string, string>;
+  white: string;
+  black: string;
```

Comment:
Just curious why white and black aren't also `Record<string, string>`?

Reply:
They just have the color value, no color scales.